static/app/components/events/interfaces/crashContent/stackTrace/platformIcon.tsx

```diff
@@ -18,7 +18,7 @@ function StacktracePlatformIcon({platform}: Props) {
 
 const StyledPlatformIcon = styled(PlatformIcon)`
   position: absolute;
-  top: -1px;
+  top: 0;
```

Comment:
Because this was moved outside of the panel, this was necessary to get it to align properly.